examples/static-site/Caddyfile

```diff
@@ -0,0 +1,5 @@
+0.0.0.0:8080 {
+        root /var/www/public
+        log stdout
+        gzip
```

Comment:
Apparently Caddy allows `gizip` directive to avoid compressing certain urls https://caddyserver.com/docs/gzip

But I guess we can live without that if this is just a guideline, as @erizocosmico says